### DISCUSSION TITLE
Size of GUI on 4k monitor (Wayland)

### DISCUSSION THREAD

**Discussion by nicks-boson** [LinSoftWin/Photoshop-CC2022-Linux#70] [Q&A]
Hi, anybody know if there is a way to make the scale of the UI bigger?

**Comment by MiMillieuh** [comment]
> Hi, anybody know if there is a way to make the scale of the UI bigger?

Maybe using wine with Wayland patches... x11 doesn't support that.

**Reply by ikim23code** [reply]
Did you find any solution for the DPI scaling? Thanks

**Reply by MiMillieuh** [reply]
> Did you find any solution for the DPI scaling? Thanks

Enable it in mutter settings if you're on gnome but for other desktop I have no idea...

**Reply by ikim23code** [reply]
Thanks for the response. Unfortunately, I'm using Manjaro KDE Plasma.

**Reply by MiMillieuh** [reply]
> Thanks for the response. Unfortunately, I'm using Manjaro KDE Plasma.

Maybe there is something similar to that on KDE.

It's called fractional scaling. 
Although it might make some apps a bit blurry for X11 apps

**Reply by ikim23code** [reply]
What I could find to work is to go to: Edit - Preferences - Interface - Ui Font Size and set it to Large, also check the "Scale UI to Font" option. It will not increase the font size but it will increase the size of the icons.
